.github/jobs/configure-checks/all.bats

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/env bats
+
+load 'assert'
+
+u="domjudge-bats-user"
+
+setup_user() {
+    id -u $u || useradd $u >/dev/null
+    chown -R $u:$u ./
+}
+
+setup() {
+    setup_user
+    for shared_file in config.log confdefs.h conftest.err; do
+        chmod a+rw $shared_file || true
+    done
+    repo-install gcc g++ libcgroup-dev
+}
+
+q_configure () {
```

Comment:
What the reason for the `q_` prefix?

Reply:
I needed a prefix to signal that this is non standard. Any suggestion for a good name? Otherwise I'll go with `run_configure`.